src/wkt/src/wrappers.rs

```diff
@@ -213,17 +213,49 @@ impl crate::message::Message for Int64Value {
     }
 }
 
+use base64::{engine::general_purpose::STANDARD, Engine};
```

Comment:
nit: I think we try to avoid `use` in the middle of a module. Sometimes we add them at the beginning of functions.  Now you made me thing about whether `use` declarations affect all the code or just the code after the declaration... 🤔

Reply:
Obviously they are visible in the full module, like any other declaration.  Better move this to the top of the file.

Reply:
Moved